hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/grantapproval/ERCGrantApprovalCall.java

```diff
@@ -89,7 +90,10 @@ public PricedResult execute() {
                     : GrantApprovalTranslator.ERC_GRANT_APPROVAL_NFT
                             .getOutputs()
                             .encodeElements();
-            return gasOnly(successResult(encodedOutput, gasRequirement), status, false);
+            return gasOnly(
+                    successResult(encodedOutput, gasRequirement, (ContractCallRecordBuilder) recordBuilder),
```

Comment:
Same idea, can we just dispatch using `ContractCallRecordBuilder.class` as type?

Reply:
Thanks. Addressed.